android_jni/avifandroidjni/src/main/jni/libavif_jni.cc

```diff
@@ -74,6 +74,18 @@ bool CreateDecoderAndParse(AvifDecoderWrapper* const decoder,
     LOGE("Failed to parse AVIF image: %s.", avifResultToString(res));
     return false;
   }
+
+  avifDiagnostics diag;
+  if (!(decoder->decoder->image->transformFlags & AVIF_TRANSFORM_CLAP) ||
+      !avifCropRectConvertCleanApertureBox(
```

Comment:
Optional: It would be good to add a comment to point out how we handle invalid 'clap' properties here. Instead of failing, we ignore the 'clap' property and display the full image.

Reply:
Done.